tools/sixpack.md

```diff
@@ -0,0 +1,166 @@
+
+## Output
+
+The webtool will redirect you to a new page with a couple of tabs near the top. 
+We are primarily interested in the `Result Summary` (which you should be on to begin)
+and `Tool Output` tabs, and describe them in the following. The results 
+can be downloaded by clicking the `Download ...` button underneath the tabs. 
+
+### Result Summary
+
+This page begins by visualizing the input sequence and (optionally) its reverse compliment 
+and the codons obtained by translating the sequences in various reading frames. The following 
+are the first few lines of the page (line numbers `LX` added for reference):
+
+```
+  H  R  T  E  Q  H  K  F  G  R  N  H  M  I  T  V  H  A  L  *     F1   (L1)
+   T  G  Q  N  N  T  N  L  A  G  I  T  *  *  R  F  T  L  F  S    F2   (L2)
+    Q  D  R  T  T  Q  I  W  Q  E  S  H  D  N  G  S  R  S  L  V   F3   (L3)
+1 CACAGGACAGAACAACACAAATTTGGCAGGAATCACATGATAACGGTTCACGCTCTTTAG 60     (L4)
+  ----:----|----:----|----:----|----:----|----:----|----:----|        (L5)
+1 GTGTCCTGTCTTGTTGTGTTTAAACCGTCCTTAGTGTACTATTGCCAAGTGCGAGAAATC 60     (L6)
+   X  L  V  S  C  C  L  N  P  L  F  *  M  I  V  T  *  A  R  *    F6   (L7)
+  X  C  S  L  V  V  C  I  Q  C  S  D  C  S  L  P  E  R  E  K     F5   (L8)
+    V  P  C  F  L  V  F  K  A  P  I  V  H  Y  R  N  V  S  K  L   F4   (L9)
+```
+
+The sequence is shown in `L4` and its reverse compliment in `L6`. The flanking numbers `1` and `60` 
+indicate that we are looking at the 1-st to 60-th (inclusive) bps of the input sequence. Each row
+above and below gives the amino acids (AAs) obtained by translating the input sequence in a specific reading frame, 
+labelled `FX` for "frame `X`".
+`F1-3` correspond to the forward sequence while `F4-6` correspond to the reverse-compliment.
+Each AA is aligned with the first nucleotide of the codon from which it was translated. For example, the
+first AA `H` of `F1` was produced from the `CAC` codon, and the fourth AA `N` of `L2` was 
+produced from `AAC`. We clarify this with bars: 
+
+```
+  H  R  T  E  Q  H  K  F  G  R  N  H  M  I  T  V  H  A  L  *     F1   (L1)
+  |T  G  Q  N  N  T  N  L  A  G  I  T  *  *  R  F  T  L  F  S    F2   (L1)
+  |         |                                                         (L3 omitted) 
+1 CACAGGACAGAACAACACAAATTTGGCAGGAATCACATGATAACGGTTCACGCTCTTTAG 60     (L4)
+```
+
+Note that a `*` denotes a stop codon. For `F4-6`, care must be taken since these 
+are translations of the sequence's reverse compliment. Hence, `L6-9` should be read from
+right to left for all intents and purposes. For example, the `S` in the following is produced by the 
+codon `TCC`, not `CCT` or `TGT`:
+
+```
+1 GTGTCCTGTCTTGTTGTGTTTAAACCGTCCTTAGTGTACTATTGCCAAGTGCGAGAAATC 60     (L6)
+        |                                                             (L7 omitted)
+  X  C  S  L  V  V  C  I  Q  C  S  D  C  S  L  P  E  R  E  K     F5   (L8)
+```
+
+Finally, at the very end of the page, there will also be some summary statistics that count
```

Comment:
So is more ORFs better or less ORFs better? How are you interpreting these values?

Reply:
OK consider the example with SARS-CoV-2 in class:

![SC2 ORF Calling](https://github.com/ababaian/VIRUSxDISCVRY/assets/9220907/a0b5781c-db72-4f21-89c2-ddb3ff7c2a55)

This won't be in the literature / textbook, you have to assess it yourself. What does it mean if a particular frame has very few ORFs, and what does it mean if a frame has a very high number of ORFs. How does ORF length vary with each of these?

Reply:
If a frame has few (resp., many) ORFs, then the ORF lengths in that frame will be larger (resp., smaller). From my understanding, longer ORFs are favoured in analyses. However, I feel hesitant to use this analysis as it fails when multiple long ORFs exist in different frames. Instead, I would prefer to sort the entries in `Tool Output` by length.